csunplugged/topics/management/commands/_ProgrammingChallengesLoader.py

```diff
@@ -6,6 +6,9 @@
 from utils.errors.MissingRequiredFieldError import MissingRequiredFieldError
 from utils.TranslatableModelLoader import TranslatableModelLoader
 
+from pluggingitin.models import (
```

Comment:
Just a note for later: Using a loader for loading data into two different Django applications is possibly a sign we need to move all programming content to the plugging it in application.